### PR TITLE
Improve Code_Changes implementation and metric description

### PR DESCRIPTION
**Pull request by jgbarah**
This is a more elaborate case of how a metric description, and its reference implementation, can be.

I've included content to complete the description of the metric, with the parameters needed to understand it, and the specific stuff relevant for the case of git.

I've improved the notebook for implementing it.

### CODE REVIEW

File: implementations/README.md [hunk 1]
```diff
@@ -22,16 +22,24 @@ $ pip install perceval
...
+```
+$ jupyter notebook
 ```
 
-Of course, instead of `Code_Commits.ipynb` you can load any other notebook.
 This will launch the Jupyter kernel, and will open your default browser
-with the notebook loaded. More detailed instructions can be found in
-[Introducing the Notebook Server’s Command Line Options](Introducing the Notebook Server’s Command Line Options).
+with the directory with all the notebooks loaded.
+Click on the notebook you want to dun, and you're ready to go.
```
Comment (germonprez) [inline implementations/README.md]: run? not dun.

Comment (jgbarah) [reply implementations/README.md]: Ooops! Thx for noticing!

Comment (jgbarah) [reply implementations/README.md]: Fixed!

File: metrics/Code_Changes.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+(merging ids corresponding to the same author).
+
+* By groups of actors (employer, gender...). Requires actor grouping,
+and likely, actor merging.
+
+## Visualizations
+
+Some useful visualizations are:
+
+* Count per month over time
```
Comment (GeorgLink) [inline metrics/Code_Changes.md]: For visualizations, would it make sense to suggest a graph type? Maybe a column chart would be appropriate.

File: metrics/Code_Changes.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+* Include merge commits. Booelan. Default: True.
+
+    Merge commits are
+    those which merge a branch, and in some cases are not considered as
+    reflecting a coding activity
+
+* Include empty commits. Boolean. Default: True.
+
+    Empty commits are
+    those with do not touch files, and in some cases are not considered as
```
Comment (GeorgLink) [inline metrics/Code_Changes.md]: word: those _with_ do -> those **which** do

File: metrics/Code_Changes.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+Usual aggregators are:
+
+* Count. Total number of changes during the period.
+
+## Specific description: git
+
+In the cases of git, a code change is defined as the part of
+a git commit that "touches" a source file.
+That is, for each git commit, only the part related to
+deletions or additions of lines to files that are considered source code
```
Comment (GeorgLink) [inline metrics/Code_Changes.md]: I seem to remember that we had a definition for _source code_ or discussion about what is considered _source code_ somewhere. How about linking to it to provide clarity for implementers?

File: focus_areas/code_development.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ Goal **Activity**:
 
 * Question **Changes**: How many changes are happening to the code base, during a certain time period? 
 
-  * Metric **Code_Changes_No**(Period): Number of changes to the code base (see [Code_Changes][../metrics/Code_Changes.md])
+  * Metric **Code_Changes_No**(Period): Number of changes to the code base (see [Code_Changes](../metrics/Code_Changes.md])
```
Comment (GeorgLink) [inline focus_areas/code_development.md]: line 29: The closing square bracket `]` after ".md" seems out of place

File: metrics/Code_Changes.md (hunk 2 above)
Comment (jgbarah) [reply metrics/Code_Changes.md]: Good idea! I will suggest column chart.

File: metrics/Code_Changes.md (hunk 3 above)
Comment (jgbarah) [reply metrics/Code_Changes.md]: Fixed, thanks.

File: metrics/Code_Changes.md (hunk 4 above)
Comment (jgbarah) [reply metrics/Code_Changes.md]: We have some discussion about this in #72, but I don't think it is conclusive enough... I was planning to work on it on a further version of the implementation, to at least give some clues of how that could be implemented....

File: focus_areas/code_development.md (hunk 5 above)
Comment (jgbarah) [reply focus_areas/code_development.md]: Of course you're right. Thanks!

**Comment by jgbarah** [comment]
Thanks for the positive reviews. I'm merging after addressing your comments.

**Pull request merged**